prisma/schema.prisma

```diff
@@ -1,11 +1,69 @@
 // This is your Prisma schema file,
 // learn more about it in the docs: https://pris.ly/d/prisma-schema
+//start generating prisma clients. start building api routes, server network request to perform databse operations (this is how ui interacts with database)
 
 generator client {
   provider = "prisma-client-js"
 }
 
 datasource db {
-  provider = "postgresql"
+  provider = "sqlite"
+  //url      = env("PRISMA_DB_URL") //one youtube vide put "file:./dev/db"
   url      = env("PRISMA_DB_URL")
 }
+
+//get url from .env file?
+
+//to run migration, run npx prisma migrate dev, and apply to database
+
+model User {
+  id    String @id @default(uuid()) //@id makes it the primary key. cuid makes random id. uuid is unversial unique id
+  email String @unique
+
+  @@map("users")
+}
+
+model Project {
+  id          String @id @default(uuid())
+  title       String @default("")
+  description String @default("")
+  partnerInfo String @default("") @map("partner_info") //in the future support partner info being tracked, maybe separate database
```

Comment:
Lets go ahead and make a partner table:

Partner
- name
- contact_name
- contact_email
- Projects Project[]

and lets link it to the Project table (one to many from project to partner)